pkgs/applications/audio/sublime-music/default.nix

```diff
@@ -53,9 +61,12 @@ python3Packages.buildPythonApplication rec {
   # https://github.com/NixOS/nixpkgs/issues/56943
   strictDeps = false;
 
-  # no tests
-  doCheck = false;
-  pythonImportsCheck = [ "sublime_music" ];
```

Comment:
Why did you remove the importsCheck?

Reply:
I am now using the upstream unit tests directly, rather than relying on the imports check. https://nixos.org/manual/nixpkgs/unstable/#using-pythonimportscheck says that it's "highly preferred" to use unit tests over the `pythonImportsCheck`.

If you want I can add it back, but it's redundant considering that the unit tests import the module and much more.

Reply:
checkInput can contain extra packages which can hide import errors. It doesn't hurt to add a pythonImportsCheck and adds some safety to not miss depedencies.